Index: main.cpp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- main.cpp	(date 1574575282695)
+++ main.cpp	(date 1574575282695)
@@ -0,0 +1,6 @@
+#include <iostream>
+
+int main() {
+    std::cout << "Hello, World!" << std::endl;
+    return 0;
+}
\ No newline at end of file
Index: CMakeLists.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- CMakeLists.txt	(date 1574575282572)
+++ CMakeLists.txt	(date 1574575282572)
@@ -0,0 +1,6 @@
+cmake_minimum_required(VERSION 3.15)
+project(data_structure_final)
+
+set(CMAKE_CXX_STANDARD 17)
+
+add_executable(data_structure_final main.cpp)
\ No newline at end of file
